packages/shared/web/src/dashboard-view/hooks/use-borrow-fixed-dashboard.tsx

```diff
@@ -0,0 +1,56 @@
+import { formatNumberAsAbbr } from '@notional-finance/helpers';
+import { useAllMarkets } from '@notional-finance/notionable-hooks';
+import { Network, PRODUCTS } from '@notional-finance/util';
+import { defineMessage } from 'react-intl';
+import { useHistory } from 'react-router';
+
+export const useBorrowFixedDashboard = (network: Network) => {
+  const {
+    yields: { fCashBorrow },
+  } = useAllMarkets(network);
+  const history = useHistory();
+  const tokenObj = {};
+
+  const allData = fCashBorrow
+    .map((y) => {
+      return {
+        ...y,
+        symbol: y.underlying.symbol,
+        title: y.underlying.symbol,
+        subTitle: `TVL: ${y.tvl ? formatNumberAsAbbr(y.tvl.toFloat(), 0) : 0}`,
+        hasPosition: false,
+        apySubTitle: defineMessage({
+          defaultMessage: `AS LOW AS`,
+          description: 'subtitle',
+        }),
+        apy: y.totalAPY,
+        routeCallback: () =>
+          history.push(
+            `/${PRODUCTS.BORROW_FIXED}/${network}/${y.underlying.symbol}`
+          ),
+      };
+    })
+    .sort((a, b) => a.apy - b.apy)
+    .filter((data) => {
```

Comment:
I think we will probably want to sort by TVL initially and that will break this filtering, so you can use a `reduce` instead here. Something like this:

```
const allData = Object.values(fCashBorrow.reduce((acc, y) => {
   if (acc[y.underlying.symbol] === undefined || acc[y.underlying.symbol].apy > y.totalAPY) {
       acc[y.underlying.symbol] = { obj }
   }
}, {})).sort((a, b) => b.tvl - a.tvl)
```